### PR TITLE
[docs] Cloud Whats New

### PR DESCRIPTION
**Pull request by ddhodge**
Add new features for Nov 18 release

https://deploy-preview-10640--infallible-bardeen-164bc9.netlify.app/latest/yugabyte-cloud/release-notes/

### CODE REVIEW

File: docs/content/latest/yugabyte-cloud/release-notes.md [hunk 1]
```diff
@@ -15,6 +15,17 @@ showAsideToc: true
 
 ## Releases
 
+### November 18, 2021
+
+#### New features
+
+- Monitor additional performance metrics for your cluster to ensure the cluster configuration matches its performance requirements using the cluster **Performance Metrics** tab. New metrics include YSQL and YCQL operations per second, YSQL and YCQL latency, network bytes per second, and more.
```
Comment (lizayugabyte) [inline docs/content/latest/yugabyte-cloud/release-notes.md:22]: I think it's always better when list items consistently start with a noun or gerund in release notes. E.g: 
Additional performance metrics for a cluster ...

File: docs/content/latest/yugabyte-cloud/release-notes.md [hunk 2]
```diff
@@ -15,6 +15,17 @@ showAsideToc: true
...
+- Review running queries on your cluster using the **Live Queries** on the cluster **Performance** tab. Use this information to visually identify relevant database operations and evaluate query execution times.
```
Comment (lizayugabyte) [inline docs/content/latest/yugabyte-cloud/release-notes.md:23]: Suggestion:
Ability to review running queries on a cluster ...

File: docs/content/latest/yugabyte-cloud/release-notes.md [hunk 3]
```diff
@@ -15,6 +15,17 @@ showAsideToc: true
...
+- Review slow YSQL queries on your cluster using the **YSQL Slow Queries** on the cluster **Performance** tab. You can use this information to identify slower running database operations, look at query execution times over time, and discover potential queries for tuning.
```
Comment (lizayugabyte) [inline docs/content/latest/yugabyte-cloud/release-notes.md:24]: Suggestion:
Ability to review slow YSQL queries ...

File: docs/content/latest/yugabyte-cloud/release-notes.md [hunk 4]
```diff
@@ -15,6 +15,17 @@ showAsideToc: true
...
+- Audit what's being done on your cloud using the **Activity** tab on the **Admin** page. The tab provides a running audit of activity on your cloud, including backups, cluster creation and modification, changes to cloud users and their roles, changes to the billing profile, and allow lists.
```
Comment (lizayugabyte) [inline docs/content/latest/yugabyte-cloud/release-notes.md:25]: Suggestion:
Ability to audit ...
or
Support for auditing ...

File: docs/content/latest/yugabyte-cloud/release-notes.md [hunk 5]
```diff
@@ -15,6 +15,17 @@ showAsideToc: true
...
+- You can now have multiple Admin users on your cloud account, and Admin users can change the role of existing users. You can also invite multiple users to your cloud at once, and assign them a role (Developer or Admin) when you invite them. You manage cloud users using the **Users** tab on the **Admin** page.
```
Comment (lizayugabyte) [inline docs/content/latest/yugabyte-cloud/release-notes.md:26]: Suggestion:
Support for multiple Admin users on a cloud account ...

File: docs/content/latest/yugabyte-cloud/release-notes.md [hunk 6]
```diff
@@ -15,6 +15,17 @@ showAsideToc: true
...
+- The **Invoices** on the **Billing** tab include additional charts that break costs down by cluster and infrastructure (instance costs, storage, and data transfer) so that you can quickly evaluate your cloud costs.
```
Comment (lizayugabyte) [inline docs/content/latest/yugabyte-cloud/release-notes.md:27]: Additional charts on the **Invoices** on the **Billing** tab ...

**Review by lizayugabyte — COMMENTED** [review]
A few suggestions.
Otherwise LGTM

**Comment by VijiYB** [comment]
Dwight, we are not actually enabling Performance tab in this release as we have a bug in that(avg read/write showing weird numbers).
 so first 3 points should be removed. Planning to enable in next prod move. Rest all are fine.

**Pull request merged**